src/libs/data-utils.js

```diff
@@ -132,7 +139,8 @@ export class UriViewer extends Component {
             metadata.timeCreated && els.cell([els.label('Created'), els.data(Utils.makePrettyDate(metadata.timeCreated))]),
             els.cell([els.label('Updated'), els.data(Utils.makePrettyDate(metadata.updated))]),
             els.cell([els.label('md5'), els.data(metadata.md5Hash)])
-          ])
+          ]),
+          div({ style: { fontSize: 10 } }, ['* Estimated. Download cost may be higher in China or Australia.'])
```

Comment:
This should probably say 'Asia', since I believe the special rates apply to other locations besides China.

Reply:
According to google's pricing info, the non-china asian destinations are the same pricing as the rest of the world.